src/model/guilds/guild_tournament.rs

```diff
@@ -872,18 +844,18 @@ impl GuildTournament {
                 return Ok(digest);
             }
         };
-        let op = TournOp::PlayerOp(p_id, RecordResult(result.clone()));
+        let r_id = self
+            .tourn
+            .round_reg
+            .get_player_active_round(&p_id)
+            .unwrap()
```

Comment:
Should we unwrap here? What if the player does not have an active round?

Reply:
I didn't write this myself, I merely moved it around.  unwrap was used in the `Ok()` arm of the following match expression. I don't know if fixing this sits with the general theme of updating to the latest `squire_lib`. What do you think?

```
match self.tourn.apply_op(Utc::now(), op)
```